File: C/unit_2/Fb.c
#include <stdio.h>
int main()
{
  int num, rev = 0, rem;
  printf("Enter the Number : \n");
  scanf("%d", &num);
  while (num != 0)
  {
    rem = num % 10;
    rev = rev * 10 + rem;
    num /= 10;
  }
  printf("The Reverse of the number is : %d", rev);
  return 0;
}

File: Java/unit_2/Fb.java
package Java.unit_2;

import java.util.Scanner;

public class Fb {
    public static void main(String[] args) {
        long n, r = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Number : ");
        n = sc.nextInt();
        while (n != 0) {
            r *= 10;
            r += n % 10;
            n /= 10;
        }
        System.out.println("The Reverse of the Number is : " + r);
        sc.close();
    }
}
